File: src/main/java/teoriadosgrafos/excecoes/CicloNegativoException.java
package teoriadosgrafos.excecoes;

/**
 * Exceção lançada ao se deparar com um ciclo negativo dentro de um grafo ponderado
 *
 * @author <a href="https://github.com/icarohs7">Icaro D Temponi</a>
 */
public class CicloNegativoException extends RuntimeException {
	/**
	 * Instantiates a new Ciclo negativo exception.
	 *
	 * @param s s
	 */
	public CicloNegativoException( String s ) {
		super( s );
	}
}

File: src/main/java/teoriadosgrafos/excecoes/CicloNegativoException.kt
package teoriadosgrafos.excecoes

/**
 * Exceção lançada ao se deparar com um ciclo negativo dentro de um grafo ponderado
 *
 * @author [Icaro D Temponi](https://github.com/icarohs7)
 */
class CicloNegativoException(s: String) : RuntimeException(s)
